android/samples/image-based-lighting/app/build.gradle

```diff
@@ -8,7 +8,7 @@ compileMaterials {
     group 'Filament'
     description 'Compile materials'
 
-    inputDir = file("src/main/materials")
+    inputFile = file("src/main/materials/clear_coat.mat")
```

Comment:
Hmm I explicitly used a directory to compile ALL materials.

Reply:
This functionality has been restored in the latest version of my fix.